### PR TITLE
Add contributor: Jorgen Lybeck 

### PR DESCRIPTION
**Pull request by Jeyloh**
Added contributor ID together with 3 posts. More TBA! 

Also added a commit with a typo for the guide to add posts. Check it out. 

Br Jørgen from Norway

### CODE REVIEW

File: content/posts/2020-07-23-destroying-backends-authentication-intro/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: Destroying Backends with Serverless — Introduction, and Authentication
+description: "Introduction to the Amplify framework. Get set up with authentication and follow the Destroying Backends series to add a GraphQL API in only hours"
+banner: "https://miro.medium.com/max/1400/1*yXdFx3k0esisdcRcD_Gi_A.png"
+authorIds:
+  - jorgen-lybeck
+href: https://medium.com/capgemini-norway/destroying-backends-with-serverless-introduction-and-authentication-e940946c6de2
+platforms:
+  - React
+categories:
+  - Introduction
```
Comment (swaminator) [inline content/posts/2020-07-23-destroying-backends-authentication-intro/index.md]: Please remove 'Introduction'

File: content/posts/2021-07-23-destroying-backends-graphql-api/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Destroying Backends with Amplify — AppSync GraphQL API
+description: "Set up GraphQL in your Amplify application with React. Learn about gotchas while creating a full CRUD API in hours!"
+banner: "https://miro.medium.com/max/2000/1*58XROGN5bxkcw_PH1H5dlg.png"
```
Comment (swaminator) [inline content/posts/2021-07-23-destroying-backends-graphql-api/index.md]: Currently we only support image files in the repo

File: content/posts/2020-07-23-destroying-backends-authentication-intro/index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (swaminator) [inline content/posts/2020-07-23-destroying-backends-authentication-intro/index.md]: Please rename the folder to 2019-07-23-destroying-backends-authentication-intro

File: content/posts/README.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ categories:
   - Functions
 ---
 ```
-6. Upload an image at the root of your post folder (e.g. /contents/events/YYYY-MM-DD-desc/avatar.png) 
+6. Upload an image at the root of your post folder (e.g. /contents/posts/YYYY-MM-DD-desc/avatar.png)
```
Comment (swaminator) [inline content/posts/README.md]: Thanks for the fix!

File: content/posts/2021-07-23-destroying-backends-graphql-api/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (swaminator) [inline content/posts/2021-07-23-destroying-backends-graphql-api/index.md]: Please rename the folder to 2019-07-23-dest..

**Comment by swaminator** [comment]
Fixed issues and added your posts to another PR: https://github.com/aws-amplify/community/pull/142

Closing this one.

**Comment by Jeyloh** [comment]
Ops, sorry @swaminator, I went to holidays and didn't read email as consistent. Thank you very much for fixing my PR!

**Pull request closed without merging**